source/agora/test/EnrollmentManager.d

```diff
@@ -78,8 +78,7 @@ unittest
     auto first_node = network.clients[0];
 
     // Request enrollment at the height of 15
-    auto enroll = first_node.createEnrollmentData();
-    first_node.enrollValidator(enroll);
+    first_node.setRecurringEnrollment(true);
```

Comment:
Ah it was used L89, sorry... I think we can remove L89 too.

Reply:
Ah, I should have check that. Will update. Thanks.